scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -129,6 +157,151 @@ SG_ CELL_MONITOR_5_DIE_TEMPERATURE : 0|32@1+ (1,0) [0.0|120.0] "degC" DEBUG
 BO_ 129 BMS_MAX_CELL_MONITOR: 4 BMS
 SG_ MAX_CELL_MONITOR_DIE_TEMPERATURE : 0|32@1+ (1,0) [0.0|120.0] "degC" DEBUG
 
+BO_ 160 INV1_Temperature_Set_1: 8 INV1
```

Comment:
I think that some of these signals don't need to be here, things like:
1. INV1_Temperature_Set
2. INV1_Flux_ID_IQ_Info
3. INV1_Internal_Voltages
4. INV1_Internal_Voltages

But they may be useful for testing--I'm not sure inverter information the DCM would be interested in. @markomarkusi  thoughts?

Reply:
Its unclear to me from the specs which CAN messages need to be read by the DCM. If most of these signals are for debugging we should probably specify them to be DEBUG messages.